app/src/main/res/layout/activity_main.xml

```diff
@@ -160,13 +162,13 @@
         android:id="@+id/bottomNavigation"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        app:itemIconTint="@drawable/navigation_selector_color"
-        app:itemTextColor="@drawable/navigation_selector_color"
+        app:itemIconTint="@color/colorTextSecondary"
```

Comment:
This is what's causing the bottom navigation to not show the selected tab anymore. Please note the `@drawable/navigation_selector_color` is a selector and not a pure color, meaning it changes depending on the selection state. Can you please fix this?

Reply:
oops good catch

Reply:
Now it's red instead of blue :)

Reply:
now it's red on Light theme, Blue on Dark theme. wasn't this the objective? (I think it looks good matching the "selectable text" colors)